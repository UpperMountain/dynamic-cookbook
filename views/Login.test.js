import React from "react";
import LoginPage from "./Login";
import renderer from "react-test-renderer";

it("renders without exploding", () => {
  const cmp = renderer.create(<LoginPage />);
});

it("passes snapshot tests", () => {
  const cmp = renderer.create(<LoginPage />);
  expect(cmp.toJSON()).toMatchSnapshot();
});
