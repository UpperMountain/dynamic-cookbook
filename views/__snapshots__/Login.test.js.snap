// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`passes snapshot tests 1`] = `
<View
  style={
    Object {
      "backgroundColor": "transparent",
      "flex": 1,
    }
  }
>
  <ExponentLinearGradient
    colors={
      Array [
        4294946749,
        4294951840,
      ]
    }
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "height": "50%",
          },
          Object {
            "alignItems": "center",
            "flexDirection": "column",
            "justifyContent": "space-evenly",
          },
        ]
      }
    >
      <Image
        source={1}
        style={
          Object {
            "height": 200,
            "width": 200,
          }
        }
      />
    </View>
    <View
      style={
        Array [
          Object {
            "height": "40%",
          },
          Object {
            "alignItems": "center",
            "flexDirection": "column",
            "justifyContent": "space-evenly",
          },
        ]
      }
    >
      <View>
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderColor": "white",
              "borderRadius": 72,
              "borderWidth": 3,
              "flexDirection": "row",
              "height": 56,
              "justifyContent": "center",
              "opacity": 1,
              "width": 315,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "white",
                "flexDirection": "row",
                "fontFamily": "raleway-bold",
                "fontSize": 18,
                "lineHeight": 32,
              }
            }
          >
            Login
          </Text>
        </View>
      </View>
      <View>
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "white",
              "borderColor": "white",
              "borderRadius": 72,
              "borderWidth": 3,
              "flexDirection": "row",
              "height": 56,
              "justifyContent": "center",
              "opacity": 1,
              "width": 315,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#FFC3A0",
                "flexDirection": "row",
                "fontFamily": "raleway-bold",
                "fontSize": 18,
                "lineHeight": 32,
              }
            }
          >
            Register
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "paddingLeft": "20%",
            "paddingRight": "20%",
            "width": "100%",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "backgroundColor": "#fff",
                "flex": 1,
                "height": 1,
              },
            ]
          }
        />
        <Text
          style={
            Object {
              "color": "#fff",
              "flex": 0.3,
              "textAlign": "center",
            }
          }
        >
          or
        </Text>
        <View
          style={
            Array [
              Object {
                "backgroundColor": "#fff",
                "flex": 1,
                "height": 1,
              },
            ]
          }
        />
      </View>
      <View>
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#DB4437",
              "borderColor": "#DB4437",
              "borderRadius": 72,
              "borderWidth": 3,
              "flexDirection": "row",
              "height": 56,
              "justifyContent": "center",
              "opacity": 1,
              "width": 315,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#fff",
                "flexDirection": "row",
                "fontFamily": "raleway-bold",
                "fontSize": 18,
                "lineHeight": 32,
              }
            }
          >
            Sign in With Google
          </Text>
        </View>
      </View>
      <View>
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#3C5A99",
              "borderColor": "#3C5A99",
              "borderRadius": 72,
              "borderWidth": 3,
              "flexDirection": "row",
              "height": 56,
              "justifyContent": "center",
              "opacity": 1,
              "width": 315,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#fff",
                "flexDirection": "row",
                "fontFamily": "raleway-bold",
                "fontSize": 18,
                "lineHeight": 32,
              }
            }
          >
            Continue With Facebook
          </Text>
        </View>
      </View>
    </View>
  </ExponentLinearGradient>
</View>
`;
